### DISCUSSION TITLE
GUI not coming up after initial setup

### DISCUSSION THREAD

**Discussion by GauchoOG** [gboudreau/Greyhole#306] [General]
Hello! After finishing the GUI setup and restarting my linux , the GUI is not coming up anymore. SQL is started, SAMBA is started, Grayhole is started... I can't see anything obvious on the logs. Can you help me? Thanks!

**Comment by gboudreau** [comment]
You can get more info here on how to run the Web UI on boot : https://github.com/gboudreau/Greyhole/wiki/Admin-Web-UI
